seqr/views/apis/data_manager_api_tests.py

```diff
@@ -573,12 +573,10 @@ def test_update_rna_seq(self, mock_logger, mock_open, mock_load_uploaded_file, m
             'Parsed 1 RNA-seq samples',
             'Attempted data loading for 0 RNA-seq samples in the following 0 projects: ',
         ]
```

Comment:
It could be better to separate the tests for the function `load_rna_seq` and `update_rna_seq`.

Reply:
This file has one command. This test tests what happens when you call that command. The command does 2 things in a row so it tests that both things are done. Nowhere in our repo do we try to split the test for a single successful call into different tests